src/cpp/common/alts_context.h

```diff
@@ -0,0 +1,36 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef GRPC_INTERNAL_CPP_COMMON_ALTS_CONTEXT_H
+#define GRPC_INTERNAL_CPP_COMMON_ALTS_CONTEXT_H
+
+#include <grpcpp/security/auth_context.h>
+
+#include "src/core/lib/security/context/security_context.h"
+#include "src/proto/grpc/gcp/altscontext.pb.h"
+
+namespace grpc {
+// GetAltsContextFromAuthContext helps to get the AltsContext from AuthContext.
+// Please make sure the underlying protocol is ALTS before calling this
+// function, otherwise a nullptr will be returned.
+std::unique_ptr<gcp::AltsContext> GetAltsContextFromAuthContext(
+    const AuthContext& auth_context);
```

Comment:
Note if this is considered an API for all users, it should go to somewhere under `include/grpcpp` ... If it is just for some internal library etc, then this is fine.

Reply:
Thanks for pointing this out! I have moved this to "include/grpcpp/alts_context.h". Could you please take a second look on if this looks good to you?